packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -620,6 +620,14 @@ class CustomScrollView extends ScrollView {
 
 /// A [ScrollView] that uses a single child layout model.
 ///
+/// {@template flutter.widgets.BoxScroll.scrollBehaviour}
+/// ## Scrolling Behaviour
+///
+/// [ScrollBehavior] cannot be provided because it is only exposed
+/// in widgets where the scrolling is hard-coded due to scrollbars and
+/// overscroll indicators, or widget-specific [ScrollPhysics].
```

Comment:
This kind of assumes the user is aware that other types of scrollable widgets can have a ScrollBehavior provided. If I do not know this, I read this and wonder, what do you mean scroll behavior?

Also, I realized mentioning physics here doesn't make sense since `physics' is a parameter that is already available.

What about something like...

```suggestion
/// [ScrollView]s are often decorated with [Scrollbar]s and overscroll indicators, which are managed by the inherited [ScrollBehavior]. Placing a [ScrollConfiguration] above a ScrollView can modify these behaviors for that ScrollView, or can be managed app-wide by providing a ScrollBehavior to [MaterialApp.scrollBehavior] or [CupertinoApp.scrollBehavior].
```

Feel free to adjust. Thanks for using a template!